modules/swagger-codegen/src/main/resources/nodejs-google-cloud-functions/index.mustache

```diff
@@ -0,0 +1,44 @@
+'use strict';
```

Comment:
@jmuk we'll move this file to nodejs/google-cloud-functions/index.mustache (and update the code accordingly). so that all nodejs (server) related files are under the folder.

Are you OK with the proposed change?